src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.net46.cs

```diff
@@ -8,17 +8,17 @@ namespace System.IO
 {
     public static class FileSystemAclExtensions
     {
-        public static DirectorySecurity GetAccessControl(DirectoryInfo directoryInfo)
+        public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo)
```

Comment:
Not 100% sure what the actual guideline is but I normally do a null check to avoid an NullReferenceException